bukuserver/templates/bukuserver/lib.html

```diff
@@ -35,7 +35,7 @@
   <script>
     $('.admin-form [name=fetch]').remove();
     $('.admin-form').append(
-      $(`<div class="form-group"><label style="display: block"><span class="col-md-2 text-right">{{ _gettext('Fetch') }} &nbsp; </span>`
+      $(`<div class="form-group"><label style="display: block"><span class="col-md-2 text-right">{{ _('Fetch') }} &nbsp; </span>`
```

Comment:
Flask-Admin doesn't have this translation key (or at least an appropriate translation for it); switching to using app translations.